src/wamr/WAMRWasmModule.cpp

```diff
@@ -69,35 +75,38 @@ namespace wasm {
 
         // Load the wasm file
         storage::FileLoader &functionLoader = storage::getFileLoader();
-        std::vector<uint8_t> aotFileBytes = functionLoader.loadFunctionWamrAotFile(msg);
+#if(WAMR_EXECUTION_MODE_INTERP)
+        wasmBytes = functionLoader.loadFunctionWasm(msg);
+#else
+        std::vector<uint8_t> wasmBytes = functionLoader.loadFunctionWamrAotFile(msg);
+#endif
```

Comment:
See comment above, why can't this be a local variable in `WAMR_EXECUTION_MODE_INTERP` too?